### PR TITLE
Fixed the loader issue

### PR DESCRIPTION
**Pull request by sudeeptarlekar**
## Notion card
NA

## Summary
After failing wise api loading keeps spinning. Fixed the loader issue

### CODE REVIEW

File: app/javascript/src/components/Profile/BankAccountDetails/BankInfo.tsx [hunk 1]
```diff
@@ -6,28 +6,35 @@ const BankInfo = ({
...
+    try {
+      setIsLoading(true);
+      const response = await wiseApi.fetchRecipient(recipientId);
+      const name = response.data["accountHolderName"].split(" ");
+      setRecipientDetails(response.data);
+      setFirstName(name.shift());
+      setLastName(name.join(" "));
+      setIsUpdate(true);
+    } catch (error) {
+      console.error(error);
```
Comment (apoorv-mishra) [inline app/javascript/src/components/Profile/BankAccountDetails/BankInfo.tsx:32]: We can replace `console.error` with [this](https://github.com/saeloun/miru-web/blob/7fc43f458ee085e71d26f2784d73090ced7d8680/app/javascript/src/components/Invoices/Invoice/index.tsx#L24-L31) pattern.

File: app/javascript/src/components/Profile/BankAccountDetails/CurrencyDropdown.tsx [hunk 2]
```diff
@@ -11,24 +11,30 @@ const CurrencyDropdown = ({
...
+  const fetchCurrencyList = async () => {
+    try {
+      setIsLoading(true);
+      const response = await wiseApi.fetchCurrencies();
+      const list = response.data.map(currency => (
+        { label: `${currency} (${getSymbolFromCurrency(currency)})`, value: currency }
+      ));
+      setCurrencies(list);
+    } catch (error) {
+      console.error(error);
```
Comment (apoorv-mishra) [inline app/javascript/src/components/Profile/BankAccountDetails/CurrencyDropdown.tsx:32]: > We can replace console.error with [this](https://github.com/saeloun/miru-web/blob/7fc43f458ee085e71d26f2784d73090ced7d8680/app/javascript/src/components/Invoices/Invoice/index.tsx#L24-L31) pattern.

ditto!

File: app/javascript/src/components/Profile/BankAccountDetails/index.tsx [hunk 3]
```diff
@@ -25,15 +25,21 @@ const BankAccountDetails = () => {
...
+    fetchProfileDetails();
   }, []);
 
+  const fetchProfileDetails = async () => {
+    try {
+      setIsLoading(true);
+      const response = await profilesApi.get();
+      setBillingDetails(response.data);
+    } catch (error) {
+      console.error(error);
```
Comment (apoorv-mishra) [inline app/javascript/src/components/Profile/BankAccountDetails/index.tsx:37]: > We can replace console.error with [this](https://github.com/saeloun/miru-web/blob/7fc43f458ee085e71d26f2784d73090ced7d8680/app/javascript/src/components/Invoices/Invoice/index.tsx#L24-L31) pattern.

ditto!

File: app/javascript/src/components/Profile/BankAccountDetails/index.tsx [hunk 4]
```diff
@@ -62,29 +68,23 @@ const BankAccountDetails = () => {
...
+      const data = response.data;
+      const validRecipientDetails = bankFieldValidationRequirements(data, isUpdate);
+      setValidRecipientDetails(validRecipientDetails);
+      if (!isUpdate) {
+        setRecipientDetails({ details: { address: {} } });
+      }
+      const fields = data.map(requirement => separateAddressFields(requirement));
+      setBankRequirements(fields);
+    } catch (error) {
+      console.error(error);
```
Comment (apoorv-mishra) [inline app/javascript/src/components/Profile/BankAccountDetails/index.tsx:84]: > We can replace console.error with [this](https://github.com/saeloun/miru-web/blob/7fc43f458ee085e71d26f2784d73090ced7d8680/app/javascript/src/components/Invoices/Invoice/index.tsx#L24-L31) pattern.

ditto!

**Review by apoorv1316 — APPROVED** [review]

File: app/javascript/src/components/Profile/BankAccountDetails/BankInfo.tsx (hunk 1 above)
Comment (sudeeptarlekar) [reply app/javascript/src/components/Profile/BankAccountDetails/BankInfo.tsx:32]: yeah will change it, but wondering what are we gonna do with the status?

**Review by apoorv-mishra — APPROVED** [review]
LGTM!

**Comment by supriya3105** [comment]
@sudeeptarlekar Please resolve the conflicts on this PR and merge it.

**Comment by sudeeptarlekar** [comment]
@supriya3105 rebased with develop

**Pull request merged**